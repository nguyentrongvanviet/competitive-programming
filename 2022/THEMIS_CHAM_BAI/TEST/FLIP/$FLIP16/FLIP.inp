80 828
842106 927291
156034 360769
159139 672738
455102 998910
721855 453843
159019 966136
553621 243844
883874 416277
504714 913624
71929 322531
993026 711519
96062 274198
995131 434527
326021 164161
158104 288475
142576 606194
980801 475310
679756 827186
846507 224471
268118 994504
536871 129002
40936 368494
625461 718360
184146 526755
150188 541506
658893 18133
182240 126504
400728 687828
98614 976709
746805 24903
285380 346210
781725 44872
656161 205620
352442 783727
744009 51432
969896 254740
242588 618801
818628 370744
455868 887563
577082 315079
298435 161062
62133 522675
914548 641495
381693 816587
643037 392373
980337 425073
872735 105871
80775 751078
195957 801510
724384 256030
533719 734124
73761 702644
724159 881851
292215 260190
962773 818810
961650 459270
840872 819515
311385 545027
392161 170322
853362 586826
636430 88656
697358 198753
718380 208628
529215 491184
286575 648269
298991 981704
809544 340749
703953 968210
322579 741584
373896 653286
718366 651963
499256 101187
915522 200915
997836 113334
126076 429683
481942 945511
752566 253584
271678 365998
358240 689729
979900 892698
718768
559415
24673
92836
767429
712159
880799
701414
388822
197599
923539
150428
267332
260166
11345
535954
543001
518929
646198
235526
723264
431281
386869
439390
61782
840854
557894
275043
597729
68298
107301
690323
559337
887340
666900
548004
881811
123997
345562
79226
293590
606635
685298
317568
383225
235133
316915
582384
906962
585481
576429
249418
655406
670338
272627
149423
262290
88534
943305
417052
381280
108261
751675
589743
847894
278940
902737
720249
626510
659721
942622
949528
735457
419014
726069
651872
790630
265907
464596
911154
388242
315618
485798
372594
725189
3076
843593
66129
859487
864462
45105
588570
639652
447342
526661
112090
77740
429664
750519
348272
414164
624325
767578
261572
70418
569687
526911
757291
728462
283902
259226
859183
141023
663277
496601
427147
823590
8144
912227
918229
114295
411549
652199
495319
55649
758312
550805
506057
641116
555039
622876
662919
784641
613185
381979
118276
363149
185512
132611
222138
880032
673203
231693
884681
192513
830967
758590
43337
577740
413894
7998
126556
113725
769837
325100
123635
227341
363487
334622
815892
472631
371748
656143
379702
626896
90588
60418
906336
837182
51431
396531
751213
283323
145300
708789
385441
324780
303780
851771
93635
239874
729604
851568
375190
155461
391025
338938
684868
46215
920090
427467
423734
278153
770194
665815
140062
224003
216471
661755
903234
557358
595285
298231
893508
171832
831981
706058
876190
192979
370155
499431
582657
756189
142290
805719
971957
677249
243963
720912
22465
459315
418981
469277
212974
357851
323981
197621
313670
888951
162711
769176
948383
32139
975335
530044
112922
688006
328503
323968
798298
354799
59240
91756
869051
731458
152555
541443
945913
744870
907745
761043
168837
532944
813960
762502
137127
730489
211427
338260
172428
706109
804260
431841
177944
833141
259598
79261
180547
507190
557034
855650
560330
635769
999397
654168
114101
348672
107531
794106
956738
127423
609808
817229
854542
515665
424339
609973
654403
114914
148115
431156
336581
961080
142520
533283
167480
439233
248050
94204
422519
509897
290205
298739
940062
961607
121911
46085
877160
481015
579124
75267
804992
825683
699766
72558
519887
296324
842641
883412
370973
404818
891121
383402
65486
115233
233844
844884
993540
368722
287752
379425
312662
740884
422324
753903
961206
823946
425251
51789
37718
877990
102598
965663
249177
134635
653721
222021
423928
222533
403520
827242
119503
503141
183997
871084
95765
295735
544985
755180
264429
865149
617238
406817
674735
798666
804074
630390
464788
619744
154689
761184
12442
647713
251580
955171
317311
640292
15929
364008
447297
926407
638461
118479
431097
370297
274733
858064
363219
238173
339114
668735
702993
590869
971665
616397
519222
379569
769407
224909
989903
864494
620454
579666
7152
938784
90028
242151
652093
332622
260794
512789
686415
588862
757798
276522
38839
579074
647601
855818
795003
353779
622040
329807
967930
976000
296733
222002
858366
307841
803218
7803
188149
951279
249024
52108
47481
283780
861910
198786
562123
547482
467400
252451
974394
88925
972602
245471
702645
682358
457894
997019
696896
983356
294933
939085
878328
292657
616195
22429
47582
228508
133918
159170
49594
830374
653387
511538
527577
313664
172724
213831
850849
847612
672953
174201
501054
609491
413518
285412
756715
923342
303333
93072
289200
516362
389734
774157
975415
692283
702483
640512
451251
759030
125019
419777
380579
141052
192663
304537
403771
277367
405140
442592
659853
760716
817622
394494
742054
254185
542109
803983
864696
715595
956884
174263
514102
734815
364721
493911
391767
438115
182384
764613
443715
66251
992679
565449
76638
501312
997961
840900
531700
309809
291015
989209
213957
720652
860223
564182
629496
793743
114084
885872
773857
12400
407579
587171
553953
279307
718050
209181
454047
832576
586225
117959
409878
534609
714319
804016
343395
485328
103301
889906
651769
496100
142688
296583
558930
458657
997746
991444
385864
54348
113379
82301
775093
400302
240553
92064
741072
408538
590699
368236
905659
122706
454856
521479
243420
401662
100662
279061
379489
246128
254452
110126
27177
223080
680063
82451
100955
450341
68745
619826
572373
120266
389124
832889
644535
189898
648064
557032
127253
17641
695834
421526
41218
714761
258201
419531
908831
84473
9248
416870
952507
336751
944776
119481
358363
434266
56534
144415
520240
178621
657479
42560
273396
28840
92925
494282
999602
408385
776592
86354
462458
718213
762192
255561
369955
588157
261829
560555
194020
242591
914468
101775
241511
28661
797036
493442
787172
43709
901226
117681
493830
365733
714449
626297
465051
324843
578942
893211
565030
739063
802858
220205
33360
121026
976542
104967
267868
828266
538527
898181
3373
89173
286458
330313
395294
10289
821134
856854
436897
884084
713698
379443
538770
868072
516139
731113
517640
937528
401982
569668
202475
981672
649752
81231
304538
433195
850286
761053
217068
207283
994999
409496
591859
70122
35449
784774
219250
721237
147112
25919
471530
536875
211363
843816
470813
201043
708668
728811
778986
102974
4037
655345
3381
534508
594989
511581
330945
256944
420120
265346
60578
792584
514206
45533
733249
998368
904622
494908
183720
413776
17459
745533
868283
195540
635842
958371
310091
119224
229627
847280
462260
869430
126748
896247
569006
394890
930656
880852
850411
715366
849665
986810
934416
928984
83405
232008
283966
563061
656691
608991
580429
217223
308890
240444
959635
675529
127152
34859
255859
563758
650823
916960
34143
397811
406753
102564
960458
189933
334456
68118
967979
180902
480501
305117
161267
274442
839989
668362
115466
102735
479365
899589
189882
245940
798950
771363
46988
361532
558171
234854
768045
899339
638289
846150
4278
