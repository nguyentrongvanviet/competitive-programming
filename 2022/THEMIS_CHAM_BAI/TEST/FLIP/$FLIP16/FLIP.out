927291 360769 159139 998910 453843 966136 243844 883874 913624 322531 993026 274198 995131 326021 288475 606194 980801 827186 846507 994504 536871 368494 625461 526755 541506 18133 182240 400728 976709 24903 346210 44872 205620 352442 51432 969896 618801 370744 887563 577082 298435 522675 914548 381693 392373 980337 872735 80775 195957 256030 533719 73761 881851 292215 962773 961650 819515 311385 392161 853362 636430 198753 208628 491184 286575 981704 340749 968210 322579 373896 718366 499256 915522 997836 429683 945511 253584 271678 358240 979900 